### PR TITLE
Add a homebrew target for easy install on macs

### PR DESCRIPTION
**Pull request by yschimke**
Ideally this would live at https://github.com/symphonyoss/homebrew-tap/jcurl.rb but I can't submit a PR to that location

With that structure it would be installable via

`brew install symphonyoss/tap/jcurl`

To test current file use `brew install jcurl.rb`

### CODE REVIEW

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@yschimke). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@yschimke). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

**Comment by yschimke** [comment]
CLA has been signed

**Comment by maoo** [comment]
@finos-cla-bot[bot] check

**Comment by finos-admin** [comment]
The cla-bot has been summoned, and re-checked this pull request!

**Comment by maoo** [comment]
@yschimke , you're now registered as FINOS Contributor. Thanks.

File: jcurl.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  version "0.9.11"
+  url "https://github.com/symphonyoss/JCurl/releases/download/jcurl-#{version}/jcurl.jar", :nounzip => true
+  sha256 "b4bb73bee7a29b28e18ea7d16f8ed4d6d86222e92da8771e9ff25ac3234acd1c"
+
+  def install
+    prefix.install "jcurl.jar"
+    (bin/"jcurl").write <<~EOS
+      #!/bin/sh
+      
+      java -jar #{prefix}/jcurl.jar
```
Comment (yschimke) [inline jcurl.rb]: need to fix an embarrassing bug

**Review by ldrozdz — APPROVED** [review]
Thanks for contributing! LGTM

**Pull request merged**